src/app/clusters/wifi-network-diagnostics-server/CodegenIntegration.cpp

```diff
@@ -59,18 +66,12 @@ constexpr bool IsAttributeEnabled(EndpointId endpointId, AttributeId attributeId
 
 } // namespace
 
+// This callback is only called for fixed endpoints.
```

Comment:
That's not true; this is called for dynamic endpoints if they're registered with the ember machinery.

Reply:
Updated the comment accordingly.